Assets/Controllers/MouseInputManager.cs

```diff
@@ -7,18 +7,23 @@ public class MouseInputManager : MonoBehaviour {
 
     [SerializeField]
     int hexLayer;
+    [SerializeField]
+    float maxTimeAllowedToDoubleClick = 0.5f;
```

Comment:
This is another one where I don't have enough understanding of the framework mechanisms to counter (or agree). I'd think a config file would let you do the same thing but maybe that is IDE specific.